test/evergreen.js

```diff
@@ -3,10 +3,52 @@
 const path = require('path');
 
 const test = require('ava');
+const proxyquire = require('proxyquire');
 
-const evergreen = require('../lib/evergreen.js');
+const inquirerMock = require('./helpers/inquirer.js');
+
+const TEST_SUBJECT = '../lib/evergreen.js';
+const constants = require('../lib/constants.js');
+
+const ZIP_PATH = 'valid zip path';
+const TENANT_NAME = 'tenant name';
+const PROJECT = 'valid project id';
+const CREDENTIALS = 'valid credentials';
+
+test.beforeEach((t) => {
+  t.context.log = console.log;
+  t.context.getTestSubject = (overrides) => {
+    overrides = overrides || {};
+
+    return proxyquire(TEST_SUBJECT, Object.assign({
+      'lodash.memoize': (fn) => fn,
+      'inquirer': inquirerMock((questions) => {
+        return Promise.resolve({
+          confirmation: true
+        });
+      }),
+      '@blinkmobile/bm-identity': function BlinkMobileIdentity () {
+        this.assumeAWSRole = () => {
+          return Promise.resolve();
+        };
```

Comment:
Shorter (although not necessarily clearer): `this.assumeAWSRole = () => Promise.resolve()`?